packages/debugger/bin/debugger.js

```diff
@@ -45,9 +45,18 @@ const args = yargs(hideBin(process.argv))
     default: process.env.NEXT_PUBLIC_HOST,
   }).argv;
 
-process.env.FARCASTER_DEVELOPER_MNEMONIC = args["farcaster-developer-mnemonic"];
-process.env.FARCASTER_DEVELOPER_ID = args["farcaster-developer-fid"];
-process.env.SIGNER_URL = args["signer-url"];
+if (args["farcaster-developer-mnemonic"]) {
+  process.env.FARCASTER_DEVELOPER_MNEMONIC =
+    args["farcaster-developer-mnemonic"];
+}
+
+if (args["farcaster-developer-fid"]) {
+  process.env.FARCASTER_DEVELOPER_ID = args["farcaster-developer-fid"];
+}
+
+if (args["signer-url"]) {
+  process.env.SIGNER_URL = args["signer-url"];
```

Comment:
there must be some serialization happening when assigning to process.env because logging out args i get the proper undefined value
```
{
  _: [],
  'farcaster-developer-mnemonic': undefined,
  fdm: undefined,
  farcasterDeveloperMnemonic: undefined,
  'farcaster-developer-fid': undefined,
  fid: undefined,
  farcasterDeveloperFid: undefined,
  'signer-url': undefined,
  signerUrl: undefined,
  port: 3010,
  p: 3010,
  url: undefined,
  u: undefined,
  '$0': 'frames'
}
```